core/src/main/kotlin/org/evomaster/core/search/algorithms/strategy/MutationOperator.kt

```diff
@@ -0,0 +1,24 @@
+package org.evomaster.core.search.algorithms.strategy
+
+import org.evomaster.core.EMConfig
+import org.evomaster.core.search.Individual
+import org.evomaster.core.search.algorithms.wts.WtsEvalIndividual
+import org.evomaster.core.search.service.Archive
+import org.evomaster.core.search.service.FitnessFunction
+import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.Sampler
+import org.evomaster.core.search.service.mutator.Mutator
+
+interface MutationOperator {
```

Comment:
is this really a mtation operator? you are also evaluating fitness, and saving to archive based on that